salt/reactors/edx/pulsar_mitx.sls

```diff
@@ -0,0 +1,8 @@
+pulsar_mitx:
+  local.slack.post_message:
+    - tgt: {{ data['id'] }}
```

Comment:
It's not so much that you're using the `local` prefix, it's the `tgt: {{ data['id'] }}` line that will cause the `post_message` to occur on the minion that triggered the event. You could set it to 
```salt
  - tgt: 'roles:master'
  - tgt_type: grain
```
(I believe, might need to double check the syntax)